Bio/PDB/MMCIFParser.py

```diff
@@ -215,7 +215,6 @@ def _build_structure(self, structure_id):
             except ValueError:
                 serial = atom_serial_list[i]
                 warnings.warn(
-                    "PDBConstructionWarning: "
```

Comment:
This seems like an unrelated change, is it only for formatting? Does it avoid any repetition?

Reply:
It would avoid some repetition in the default traceback message.